pkg/parse/parse.go

```diff
@@ -1098,3 +1100,47 @@ func SSH(sshSources []string) (map[string]*sshagent.Source, error) {
 	}
 	return parsed, nil
 }
+
+func parseIgnore(ignoreFile string) ([]string, error) {
+	var excludes []string
+	ignore, err := ioutil.ReadFile(ignoreFile)
+	if err != nil {
+		return excludes, err
+	}
+	for _, e := range strings.Split(string(ignore), "\n") {
+		if len(e) == 0 || e[0] == '#' {
+			continue
+		}
+		excludes = append(excludes, e)
+	}
+	return excludes, nil
+}
+
+func ignoreFile(contextDir string) string {
+	for _, i := range []string{".containerignore", ".dockerignore"} {
+		ignoreFile := filepath.Join(contextDir, i)
+		if _, err := os.Stat(ignoreFile); err == nil {
+			return ignoreFile
+		}
+	}
+	return ""
+}
+func ContainerIgnoreFile(path string, contextDir string) ([]string, string, error) {
```

Comment:
The arguments here are in the reverse of the order in which I think its callers expect.